domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SamplePayment.java

```diff
@@ -11,7 +10,7 @@
     private BigDecimal amount = new BigDecimal("4000");
     private String reference = "reference";
     private String dateCreated = "2010-10-12";
-    private PaymentState paymentState = new PaymentState("status", true);
```

Comment:
PaymentState is not used here, so why don't we delete it from code base? Is there any other usage?

Reply:
good point, deleting

Reply:
done